### DISCUSSION TITLE
PWA

### DISCUSSION THREAD

**Discussion by mxdpeep** [forem/forem#16679] [Feature Requests]
please add a manifest, icons and Service Worker = make the site installable as a PWA app

**Comment by citizen428** [comment]
It used to be a PWA which was changed in https://github.com/forem/forem/pull/15781. Alas, the actual referenced discussions happened in our Slack and are not publicly available. There may be public announcements about this but I didn't manage to find them just now.

**Comment by mxdpeep** [comment]
That's a pity... PWA IS THE WAY! 🛡

Filip Oščádal
CEO mxd.cz

**Comment by RihanArfan** [comment]
It's been a over a year since PWA was removed from Forem but no public reasoning why, from what I can find. Please can we have this back, as it means I can nicely have DEV on my home screen. 😀

**Reply by justinfreitag** [reply]
Bump

**Reply by j471n** [reply]
You can download the app directly from [apple store](https://apps.apple.com/us/app/forem/id1536933197) or [play store](https://play.google.com/store/apps/details?id=com.forem.android&hl=en_IN&gl=US).

**Reply by mxdpeep** [reply]
that's the hell - every week I install about 30 updates for my Android, PWA solves this problem

**Reply by RihanArfan** [reply]
Unfortunately, the Forem app isn't viable as I want dev.to on my home page and would like to use it without a massive forum switcher bar right at the bottom

**Comment by jline9507** [comment]
At its core a PWA is a website that can be installed on your device (phone, tablet[,](https://www.magiclogix.com/) PC or Mac), app icon/shortcut and all. A PWA will pull updates in the background each time a user runs it.
